base/system/userinit/lang/de-DE.rc

```diff
@@ -23,24 +31,24 @@ CAPTION "ReactOS LiveCD"
 FONT 8, "MS Shell Dlg"
 BEGIN
     CONTROL "IDB_LOGO", IDC_STARTLOGO, "Static", WS_CHILD | WS_VISIBLE | SS_OWNERDRAW, 18, 4, 290, 99
-    DEFPUSHBUTTON "Run ReactOS &Live CD", IDC_RUN, 71, 108, 175, 21
-    PUSHBUTTON "&Install ReactOS", IDC_INSTALL, 71, 136, 175, 21
+    DEFPUSHBUTTON "ReactOS &Live CD schalten", IDC_RUN, 71, 108, 175, 21
```

Comment:
Run ReactOS &Live CD has nithing to do with "schalten". I recommend "ReactOS &Live CD ausführen"

Reply:
Or "starten" instead, because it is related to the boot process.